### ISSUE TITLE
Distinguishing LinkML/NMDC tasks and MIxS specific tasks

### ISSUE THREAD

**Issue by lschriml** [GenomicsStandardsConsortium/mixs#832]
To aid our development work and workflow processes: 
    -- I am suggesting that we start to distinguish tickets/activities that are MIxS specific 
from the tasks that are LinkML/NMDC specific.
   -- This should be helpful to focus our work for the TWG and CIG. 

If the tickets/etc, are LinkML specific and are not linked to something in the MIxS, 
   let's work to distinguish these areas of work.

Cheers,
Lynn

**Comment by turbomam** [comment]
Good idea. Which issues would you use to start off the NMDC specific list?

I'm having a harder time undersigning how an issue could be LInkML-specific as opposed to MIxS specific? The current MIxS schema is expressed in the LinkML language. @sujaypatil96 , @mslarae13 and I can continue our efforts to customize the LinkML configuration in this repo and to advocate for MIxS' needs in our LInkML meetings. It would also be helpful for other CIG and or TWG participants to build up some LinkML skills , so that they can propose technical solutions to whatever gaps or inconveniences we currently have.

If LinkML hasn't turned out to be a good choice for MIxS, I would support the use of any other formal modeling language. LinkML is getting pretty good at ingesting other standards-backed formats, so NMDC could probably import whatever else MIxS chose.

**Comment by mslarae13** [comment]
I think any NMDC - GSC issues should be "NMDC has or needs this change or has received this user feedback" 
So they're issues meant to capture that this change is being requested or will need to be made in NMDC, and it is something GSC should probably also change / update. 
We have a label NMDC that is meant to capture this. I'll add to the description.
I can review the issues but any NMDC related issue is "NMDC has done, will do, needs, or has received feedback about this and GSC should know".

Hopefully that makes sense. While reviewing, if there's any issues you're specifically want me to update, comment on, or review let me know!
